exercises/queen-attack/example.js

```diff
@@ -1,12 +1,9 @@
 'use strict';
 
-module.exports = function (options) {
-  if (options === undefined) {
-    options = { white: [0, 3], black: [7, 3] };
-  }
 
+module.exports = function (options = { white: [0, 3], black: [7, 3] }) {
```

Comment:
[Default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) are a feature of ES6 and beyond.  For the moment, this JavaScript track is meant to support ES5, so we don't want to use features like this-- we reserve them for the EcmaScript track.